btcjson/cmdparse_test.go

```diff
@@ -201,7 +201,7 @@ func TestAssignFieldErrors(t *testing.T) {
 	}{
 		{
 			name: "general incompatible int -> string",
-			dest: string(0),
+			dest: "\x00",
```

Comment:
Any reason to not use `string(rune(0))` here? By the way, `"\x00"` is fine too..

Reply:
No reason, it just felt more natural to write `"\x00"` for me.